charts/kubeai/values.yaml

```diff
@@ -209,38 +209,27 @@ messaging:
   streams: []
 
 # Configure the openwebui subchart.
-openwebui:
-  fullnameOverride: "openwebui"
-  service:
-    type: ClusterIP  # ClusterIP, NodePort, LoadBalancer
-    port: 80
```

Comment:
Note to self. It's fine to remove this since this is the default: https://github.com/open-webui/helm-charts/blob/921af388489fac825a722fad2d26469b8ad94ff5/charts/open-webui/values.yaml#L197